src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import React from 'react'
+
+const ArticleCard = ({article})=>{
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+    }
+
+    console.log(article.id)
+
+
+    return (
+        <div className="ArticleCard">
+            <div className="card_body">
+                <h2 className="card_title">{article.title}</h2>
+                <h3 className="card_topic">{article.topic}</h3>
```

Comment:
Following good html semantic structure we shouldn't have more than one h3 here, do they need to be headers or can we use something else here?